public/app/core/components/AppChrome/AppChromeMenu.tsx

```diff
@@ -77,8 +77,11 @@ export function AppChromeMenu({}: Props) {
 }
 
 const getStyles = (theme: GrafanaTheme2, searchBarHidden?: boolean) => {
-  const topPosition =
-    searchBarHidden || config.featureToggles.singleTopNav ? TOP_BAR_LEVEL_HEIGHT : TOP_BAR_LEVEL_HEIGHT * 2;
+  const topPosition = config.featureToggles.singleTopNav
+    ? 0
+    : searchBarHidden
+      ? TOP_BAR_LEVEL_HEIGHT
+      : TOP_BAR_LEVEL_HEIGHT * 2;
```

Comment:
let's avoid nested ternaries again 😄 
```suggestion
  let topPosition = searchBarHidden ? TOP_BAR_LEVEL_HEIGHT : TOP_BAR_LEVEL_HEIGHT * 2;
  
  if (config.featureToggles.singleTopNav) {
    topPosition = 0;
  }
```